src/models/ProfileSchema.js

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta'
+],
+function (Okta) {
+  var _ = Okta._;
+  var { BaseSchema, SchemaProperty } = Okta.internal.models;
+
+  var ProfileSchemaPropertyCollection = SchemaProperty.Collection.extend({
+    createModelProperties: function () {
+      var modelProperties = SchemaProperty.Collection.prototype.createModelProperties.apply(this);
+      _.each(modelProperties, function (modelProperty, name) {
+        this.get(name).attributes.title = this.get(name).get('label');
```

Comment:
This is needed to show the placeholder text inside input fields.

<img width="734" alt="screen shot 2019-02-14 at 3 13 01 pm" src="https://user-images.githubusercontent.com/23267876/52823795-10066800-306b-11e9-812e-ef517a83467d.png">